applications/nrf5340_audio/broadcast_sink/Kconfig.defaults

```diff
@@ -11,6 +11,33 @@ config BT_OBSERVER
 config BT_PERIPHERAL
 	default y
 
+config BT_BUF_ACL_TX_COUNT
```

Comment:
This is ok, but consider if we can use a local prj.conf file instead of adding further to this kconfig.defaults. It adds complexity in upmerges.